network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/notification/NetPEnableReceiver.kt

```diff
@@ -68,6 +74,7 @@ class NetPEnableReceiver : BroadcastReceiver() {
     companion object {
         internal const val ACTION_NETP_ENABLE = "com.duckduckgo.networkprotection.notification.ACTION_NETP_ENABLE"
         internal const val ACTION_VPN_ENABLE = "com.duckduckgo.vpn.ACTION_VPN_ENABLE"
+        internal const val EXTRA_SNOOZED = "extra_snoozed"
```

Comment:
nit: I'd remove the EXTRA_SNOOZE and use `ACTION_VPN_SNOOZE_END` instead. It's not clear for the caller whether the extra parameter goes with that action otherwise